gradle.properties

```diff
@@ -1,9 +1,9 @@
 SONATYPE_HOST=S01
-RELEASE_SIGNING_ENABLED=true
+RELEASE_SIGNING_ENABLED=false
```

Comment:
This might create problem right and wont sign the build

Reply:
This change was made unintentionally. I will revert this
@vikram-chaitanya